src/main/java/com/work/management/web/rest/resource/BulkEmployeeRequest.java

```diff
@@ -0,0 +1,20 @@
+package com.work.management.web.rest.resource;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import java.util.List;
+import java.util.Map;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class BulkEmployeeRequest {
+
+  private List<Integer> employeeId;
```

Comment:
employeeIds...its a list..name should be plural, as it is representing multiple things.